Increased short timeout for Churn timeouts

bolt12 opened the pull request:
Closes #5072

crocodile-dentist commented:
The defaultBulkChurnInterval is 15 mins - it appears with these extended 'short' timeouts we could in principle be non-stop in churning mode, and at the same time be very slow at it.

crocodile-dentist commented:
That is true. However, IIRC the purpose was to wait a reasonable time to get most accurate counters in the logs. Even if the timeout is too short, the targets will of course be eventually met but the relevant traces will not be printed. Maybe these timeouts could be exposed to our mocking framework and we set higher values there to make sure everything works, but keep things snappy for actual use?

bolt12 commented:
Yeah, either way sounds fine, but for the test in question we don't look at the timeout values but at the traces, so the quickest way to fix is this small patch :P

bolt12 commented:
@coot  what do you think?

bolt12 commented:
for the churn test this is the haddock:

> -- | This property verifies that when nodes are running without network
> -- attenuation, decreasing numbers by churn never timeouts.

and indeed it was running with attenuations so it was flaky. I changed it to run with no attenuation and reverted the timeout changee
